DevoxxClientMobile/src/main/resources/com/devoxx/common.css

```diff
@@ -154,30 +157,49 @@
 .list-cell.schedule-cell:color {
     -fx-background-color: -track-unknown, derive(-fx-control-inner-background,-7%);
 }
-/*.list-cell.schedule-cell:track-core-platform {*/
-    /*-fx-background-color: -track-core-platform, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-emerging-languages {*/
-    /*-fx-background-color: -track-emerging-languages, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-cloud-server {*/
-    /*-fx-background-color: -track-cloud-server, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-devices {*/
-    /*-fx-background-color: -track-devices, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-client {*/
-    /*-fx-background-color: -track-client, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-devtools {*/
-    /*-fx-background-color: -track-devtools, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-devops-methodologies {*/
-    /*-fx-background-color: -track-devops-methodologies, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-unknown {*/
-    /*-fx-background-color: -track-unknown, white;*/
-/*}*/
+
+.list-cell.schedule-cell > .list-tile > StackPane {
+    -fx-min-width: 15px;
+    -fx-pref-width: 15px;
+    -fx-max-width: 15px;
+}
+
+.list-cell.schedule-cell > .list-tile > StackPane > Group > .label {
+    -fx-rotate: -90;
+    -fx-font-size: 0.35em;
+    -fx-text-fill: white;
+}
+
+.list-cell.schedule-cell.track-color0 > .list-tile > StackPane {
```

Comment:
this will just change to `.list-cell.schedule-cell:track-color0 > .list-tile > StackPane`, and so on

Reply:
With insets, and without stack pane:
```.list-cell.schedule-cell:track-color0 {
    -fx-background-color: -track-color0, white;
}```